eng/versioning.targets

```diff
@@ -57,6 +57,10 @@
         <_Parameter1>$(AssemblyName)</_Parameter1>
       </AssemblyAttribute>
 
+      <AssemblyAttribute Include="System.Resources.NeutralResourcesLanguage" Condition="'$(IsTestProject)' != 'true' and '$(IsTestSupportProject)' != 'true'">
```

Comment:
Would it make more sense to base the condition on whether or not the assembly contains resources?  I see lots of build failures due to including this in reference assemblies which weren't referencing ResourceManager.

Perhaps `'@(EmbeddedResource)' != '' and '$(IsSourceProject)' == 'true'`?

Reply:
Done!